gateway/gateway.toml

```diff
@@ -0,0 +1,9 @@
+broker = "localhost:9094"
+topic = 'topic'
+
+[messages]
+list_of_msgs = [
+    '{"aid":"aid","cid":"cid","dataType":"String","on":"1637825686000","systemOn":"1637825686000","value":"0.005"}',
```

Comment:
are the timestamps updated somewhere?

Reply:
woops, good catch!!! They aren't, I'll add that code